src/Areas/Canvases/Options/WorkbooksOptionDefinitions.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace AzureMcp.Areas.Canvases.Options;
+
+public static class WorkbooksOptionDefinitions
+{
+    public const string MessageText = "message";
+    public const string WorkbookIdText = "workbook-id";
```

Comment:
If you change this to lookup by name or id, then change this to `workbook`.  See how we do subscription

Reply:
display names of workbooks are not required to be unique, but workbook resource ids are. so could resolve by name but would potentially require de-duplication?